WordPress/Classes/Extensions/UIApplication+AppAvailability.swift

```diff
@@ -3,6 +3,7 @@ import Foundation
 enum AppScheme: String {
     case wordpress = "wordpress://"
     case wordpressMigrationV1 = "wordpressmigration+v1://"
+    case jetpack = "jetpack://"
```

Comment:
I'm using a debug build to test this PR and `isCounterpartAppInstalled` was always returning `false` when both apps were installed. Since debug builds use `jpdebug://` for JP and `wpdebug://` for WP, I updated these and the corresponding `Queried URL Schemes` so the apps can query these. Is that what you also did to test this?

Reply:
No, I didn't modify them. For **environment 1**, my setup was to have a debug build for Jetpack and a TestFlight version of WordPress. In the [apple app site association](https://wordpress.com/.well-known/apple-app-site-association), since the Jetpack app has a higher priority, it'll always be the first responder when handling external links. Plus, TestFlight builds uses release flags, so the WordPress app will have the `wordpress://` scheme registered.

As for **environment 2**, I tested both of them in isolation (e.g. test JP with WP uninstalled, and vice versa) so I kinda ignored the value of `isCounterpartAppInstalled`.

Reply:
If you're testing exclusively from the device, I think making temporary modifications like you do should also work. For reference, you can refer to the `JetpackRedirector` class:

https://github.com/wordpress-mobile/WordPress-iOS/blob/4644af904081305ab869b9a230b72ab44186fb22/WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackRedirector.swift#L10-L22

Reply:
> No, I didn't modify them. For **environment 1**, my setup was to have a debug build for Jetpack and a TestFlight version of WordPress (note that installing the App Store version also works). In the [apple app site association](https://wordpress.com/.well-known/apple-app-site-association), since the Jetpack app has a higher priority, it'll always be the first responder when handling external links. Plus, since TestFlight builds uses release flags, so the WordPress app will have the `wordpress://` scheme registered.

Gotcha! I hadn't thought to test like this because I wanted to test the effects of the PR on both apps, but this probably wasn't necessary. 

While it would be nice to avoid hard-coding the URL schemes, I think it would be very difficult. I looked into it and it might involve moving the schemes from the target build settings up to the project build settings so that either app could access them for the current release type. From there I think either app's Info.plist could still read them via a build variable like `${WPCOM_SCHEME}`.